includes/Products/Sync/Background.php

```diff
@@ -342,7 +250,8 @@ private function process_item_delete( $prefixed_retailer_id ) {
 	private function send_item_updates( array $requests ): array {
 		$facebook_catalog_id = facebook_for_woocommerce()->get_integration()->get_product_catalog_id();
 		$response            = facebook_for_woocommerce()->get_api()->send_item_updates( $facebook_catalog_id, $requests );
-		$handles             = ( isset( $response->handles ) && is_array( $response->handles ) ) ? $response->handles : [];
+		$response_handles    = $response->handles;
+		$handles             = ( isset( $response_handles ) && is_array( $response_handles ) ) ? $response_handles : array();
```

Comment:
If we are checking for the existence of property `handles` in line 254, then we should not access it in line 253. Otherwise, it would throw an `Undefined property: $handles` warning if the property is absent.
```suggestion
		$handles             = ( isset( $response->handles ) && is_array( $response->handles ) ) ? $response->handles : array();
```

Reply:
Can you see @budzanowski comment here - https://github.com/woocommerce/facebook-for-woocommerce/pull/2654#discussion_r1372755189 

> This is a tricky behaviour but it is known. The isset is not a function - it is a language construct and it works on variables only. In your case, the $response->handles is not a variable. handles is a property of the $this->response_data defined in the base class JSONResponse. You access the handles via the magic function __get from the JSONResponse. As it was mentioned isset is not able to operate on anything else than a variable and here you have a return value of a function.

> Complicated for sure :) Your code is good and you can remove the comment.